### ISSUE TITLE
[3.3.5] DB/Creature: Phantasmal Watcher missing waypoints

### ISSUE THREAD

**Issue by velinath** [TrinityCore/TrinityCore#20520]
**Description:**

Phantasmal Watchers are missing waypoints.

**Current behaviour:**

Phantasmal Watchers are spawned throughout the Dead Scar in Ghostlands.

**Expected behaviour:**

These mobs should spawn near Deatholme and wander, waypointed, up the Dead Scar up until they are close to Tranquillien. As I remember it they died or despawned once they got close to the river in the north of the zone, but unfortunately when I was trying to watch these dudes on live to make sure, other players kept getting in the way.

**Steps to reproduce the problem:**

.tele Tranquillien and wander over near the Dead Scar.

**Branch(es):**

3.3.5a

**TC rev. hash/commit:** 

0b766db

**TDB version:**  335.63 w/updates

**Operating system:** Ubuntu 16.04

**Comment by Kittnz** [comment]
Can't sniff at the moment. I guess these waypoints are really big. Prolly won't have a full waypoint on sniffs that i have right now.

**Comment by velinath** [comment]
Yeah, they're _massive_ - all the way from the far south of the zone to all the way in the north. :(

**Comment by Kittnz** [comment]
yeah, thought as much. Will have to wait until a new sniffer is available.
